lib/badge-data.js

```diff
@@ -47,12 +48,15 @@ function makeColorB(defaultColor, overrides) {
 }
 
 function setBadgeColor(badgeData, color) {
-  if (isSixHex(color)) {
+  if (isHexColor(color)) {
     badgeData.colorB = '#' + color;
     delete badgeData.colorscheme;
-  } else {
+  } else if (colorschemes[color] !== undefined){
     badgeData.colorscheme = color;
     delete badgeData.colorB;
+  } else {
+    badgeData.colorB = color;
```

Comment:
Is this the case that handles CSS colors?

There's a `validColor` function… should we be using it here? I'd suggest we reserve the `else` case for an invalid color, and make the badge red then.